.github/workflows/load-test.yml

```diff
@@ -0,0 +1,108 @@
+name: Load Test
+
+on:
+  schedule:
+    - cron: "0 0 * * 1" # Run every Monday at 00:00
```

Comment:
🌱 I'd like to see this in CI too. We also need said CI / CD to actually use the dev and QA environments we have.